gitpod-io/gitpod
# supervisor frontend

#### How to test

- Check that one can still open Theia:
https://ak-ide-host.staging.gitpod-dev.com#https://github.com/gitpod-io/go-gin-app
- Check that Code is opened if .gitpod.yml contains `ide: code`:
https://ak-ide-host.staging.gitpod-dev.com#https://github.com/akosyakov/go-gin-app
